src/controllers/UserController.js

```diff
@@ -0,0 +1,16 @@
+const db = require('../models')
+
+class UserController {
+  static async createUser(req, res) {
+    const user = req.body
+
+    try {
+      const createdUser = await db.Users.create(user)
+      return res.status(200).json(createdUser)
```

Comment:
Um último comentário, é boa prática retornar o código 201 para quando você está criando um [novo registro](https://developer.mozilla.org/pt-BR/docs/Web/HTTP/Status/201).

```suggestion
      return res.status(201).json(createdUser)
```